### PR TITLE
Fix `istioctl pc log` ztunnel pod not working

### PR DESCRIPTION
**Pull request by hanxiaop**
**Please provide a description of this PR:**
Fix https://github.com/istio/istio/issues/43588
```
./out/linux_amd64/istioctl pc log ztunnel-9v7nw.istio-system --level=ztunnel=debug
ztunnel-9v7nw.istio-system:
current log level is ztunnel=debug,info

./out/linux_amd64/istioctl pc log ztunnel-9v7nw.istio-system --level=warning
ztunnel-9v7nw.istio-system:
current log level is ztunnel=debug,warn

./out/linux_amd64/istioctl pc log ztunnel-9v7nw.istio-system --reset
2023-02-27T03:21:44.209092Z	warn	unable to get logLevel from ConfigMap istio-sidecar-injector, using default value "warning" for envoy proxies and "info" for ztunnel
ztunnel-9v7nw.istio-system:
current log level is info
```

### CODE REVIEW

File: istioctl/cmd/proxyconfig.go [hunk 1]
```diff
@@ -824,30 +831,30 @@ func logCmd() *cobra.Command {
...
+				}
+				if isZtunnelPod(pod) {
+					loggerNames[name] = Ztunnel
+				} else {
+					loggerNames[name] = Envoy
 				}
 			}
+			if err != nil {
+				return err
+			}
```
Comment (bleggett) [inline istioctl/cmd/proxyconfig.go:857]:
```suggestion

```

Does this do anything here? What error is it catching?  Or am I just misreading.

Comment (hanxiaop) [reply istioctl/cmd/proxyconfig.go]: It's not needed. Thanks

**Review by bleggett — APPROVED** [review]
LGTM

**Review by GregHanson — APPROVED** [review]

**Review by zhlsunshine — APPROVED** [review]
LGTM!

**Pull request merged**